newbank/client/ExampleClient.java

```diff
@@ -8,50 +8,55 @@
 import java.net.UnknownHostException;
 
 public class ExampleClient extends Thread{
-	
-	private Socket server;
-	private PrintWriter bankServerOut;	
-	private BufferedReader userInput;
-	private Thread bankServerResponseThread;
-	
-	public ExampleClient(String ip, int port) throws UnknownHostException, IOException {
-		server = new Socket(ip,port);
-		userInput = new BufferedReader(new InputStreamReader(System.in)); 
-		bankServerOut = new PrintWriter(server.getOutputStream(), true); 
-		
-		bankServerResponseThread = new Thread() {
-			private BufferedReader bankServerIn = new BufferedReader(new InputStreamReader(server.getInputStream())); 
-			public void run() {
-				try {
-					while(true) {
-						String response = bankServerIn.readLine();
-						System.out.println(response);
-					}
-				} catch (IOException e) {
-					e.printStackTrace();
-					return;
-				}
-			}
-		};
-		bankServerResponseThread.start();
-	}
-	
-	
-	public void run() {
-		while(true) {
-			try {
-				while(true) {
-					String command = userInput.readLine();
-					bankServerOut.println(command);
-				}				
-			} catch (IOException e) {
-				// TODO Auto-generated catch block
-				e.printStackTrace();
-			}
-		}
-	}
-	
-	public static void main(String[] args) throws UnknownHostException, IOException, InterruptedException {
-		new ExampleClient("localhost",14002).start();
-	}
```

Comment:
these changes are all just indentation, right?

Reply:
Yes, looks like changes to the indentation. I did also have a method at Line 46:
 if(command.equals("LOGOUT")){
                        bankServerOut.close();
                        server.close();
                        System.exit(0);
                    }

Which I can't see in this pull request.